docs/src/components/Navigation/SideNavigation/SideNavigation.component.tsx

```diff
@@ -1,130 +1,87 @@
 // REACT
 import * as React from 'react';
+
 // VENDOR
-import styled from 'styled-components';
-import classNames from 'classnames';
 import { Link } from 'gatsby';
-// THEME
-import { colors, sizes } from '../../../../../src/theme';
+import styled from 'styled-components';
 
-export const StyledSideNavigationElement = styled.nav`
-    min-width: 299px;
-    max-width: 299px;
-    background-color: ${colors.silver.base};
-    border-right: solid 1px ${colors.silver.dark};
-    overflow-y: auto;
-    overflow-x: hidden;
+// COMPONENTS
+import { Collapse, CollapseGroup } from '@retailmenot/anchor';
 
-    > ul {
-        > li {
-            border-bottom: solid thin ${colors.silver.dark};
-            > a {
-                font-weight: 500;
-            }
-        }
+// TODO: Change the config to allow ts extensions. This works, but tsconfig is having a snit.
+import { sections } from './sections.ts';
 
-        ul li a {
-            padding-left: ${sizes.padding.lg};
-        }
+const StyledCollapseGroup = styled(CollapseGroup)`
+    li a {
+        text-decoration: none;
 
-        &:last-of-type {
-            padding-bottom: ${sizes.padding.xl};
+        &.active {
+            background-color: #e3eef6;
```

Comment:
Perhaps it would be a good idea to import the Anchor theme and use a variable here instead of a hex?

Reply:
Although this is a good idea, ironically I don't believe the doc site conforms to the colors of RMN's theme. I will dig through it to see if I am wrong (which I'd be really happy to be in this instance).

Reply:
These colors aren't in our theme, so I added a TODO comment noting that when we have our Theme Provider in place that they should be moved into it.